### ISSUE TITLE
Landing Page Layout

### ISSUE THREAD

**Issue by rachnaraj** [REALISELab/REALISELab.github.io#6]
It feels that we have more than expected right and left margin on the landing page. Texts and content are centered aligned.
Other pages like Teams, Publications, news are left aligned (having extra right padding). If we can spread the content over the page (by keeping equal margin in all sides by to enhance the look.

For landing page it might be interesting to create sections:
1. About REALISE lab and what we do
2. Located in Montreal, Concordia Campus ( we can highlight the SE ranking of Concordia which is very good and about Montreal's vibrant culture.
3. To provide them a hyperlink to the instructions to help them craft their application (if they want to join our labs)

**Comment by DiegoEliasCosta** [comment]
HI @rachnaraj,

Those are great suggestions! @MohamedBilelBesbes, can you please work on a PR that includes the suggested changes?
Rachna, feel free to also work with Bilel on this. Once the PR is created, we can preview the website to see if we need any further changes. 

Thank you

**Comment by DiegoEliasCosta** [comment]
Also, What about having a group picture on the landing page?
We might need to take one this summer...

**Comment by rachnaraj** [comment]
Yes, having a group picture would be really cool. And yes, I can try working with Bilel on the above mentioned points. 
Thanks

**Comment by DiegoEliasCosta** [comment]
Hi @MohamedBilelBesbes  and @rachnaraj, 

Can we prioritize at least improving the layout of the page? I find it too odd that the landing page is the only one that is not nicely aligned with the other pages. 

Thank you!

**Comment by rachnaraj** [comment]
@MohamedBilelBesbes, let's share this. If you could manage the layout's alignments, I can handle the landing page's content. Let's discuss this further offline.

**Comment by MohamedBilelBesbes** [comment]
Issue fixed, please check for any remarks, I'll close the ticket otherwise

**Comment by DiegoEliasCosta** [comment]
Hi @MohamedBilelBesbes. 
I still see the landing page layout to be different than for other pages. Are you sure it is fixed?
